app/config/AppConfig.scala

```diff
@@ -30,25 +30,25 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: CTCServicesCon
   val auditingEnabled: Boolean = config.get[Boolean]("auditing.enabled")
   val graphiteHost: String     = config.get[String]("microservice.metrics.graphite.host")
```

Comment:
I think we can remove unused reference as well from this file. That I think should be reported as part of compile warnings?

Reply:
I agree we should remove unused references. However, wasn't being reported as a compile warnings interestingly, will have a look for unused references across the codebase

Reply:
Also please have a look on ```Constants.scala ```. Looks like there also we need to add type reference and need to remove some unused references.